bin/p2-dsctl/main.go

```diff
@@ -60,6 +61,14 @@ var (
 	updateMinHealth = cmdUpdate.Flag("minhealth", "The minimum health of the daemon set").String()
 	updateName      = cmdUpdate.Flag("name", "The cluster name (ie. staging, production)").String()
 	updatePodID     = cmdUpdate.Flag("pod", "The pod ID on the daemon set").String()
+
+	cmdOverrideSelector = kingpin.Command(CmdOverrideSelector, `
+		The raw selector for the daemon set represented as a string.
+		The selector string uses same syntax as the kubernetes selectors or p2-select without flags.
+		It is highly that recommended that you test it on p2-select first`,
```

Comment:
> It is highly that recommended that you test it on p2-select first

I'm glad you recognize that this operation could have huge consequences, but I don't think a warning in the help text is enough. What do you think about showing a change summary?

```
$ p2-dsctl set-selector 01-23-45-67-89 sq/datacenter=sjc1,sq/zone=web-data
Changing slug deployment from sq/datacenter=sjc1,sq/zone=web-data:
removing:    -43 hosts (awa0.sjc1.square, ...)
adding:     +339 hosts (awd139.sjc1.square, ...)
no change:     0 hosts
Continue? [yn]: 
````